app/uploaders/attachment_uploader.rb

```diff
@@ -0,0 +1,54 @@
+# encoding: utf-8
+
+class AttachmentUploader < CarrierWave::Uploader::Base
+
+  # Include RMagick or MiniMagick support:
+  # include CarrierWave::RMagick
+  # include CarrierWave::MiniMagick
+
+  # Choose what kind of storage to use for this uploader:
+  if Rails.env.production?
```

Comment:
Env branching in code is bad, m'kay?

I am pretty sure you can specify fog/carrierwave storage in env files with standard api. If not, you can do this:

# config/environments/development.rb
# config/environments/test.rb
  config.carrierwave_storage = :file

# config/environments/production.rb
  config.carrierwave_storage = :fog

# attachment_uploader.rb
class AttachmentUploader < CarrierWave::Uploader::Base